client/a8c-for-agencies/sections/purchases/licenses/license-preview/index.tsx

```diff
@@ -99,7 +96,7 @@ export default function LicensePreview( {
 	const licenseState = getLicenseState( attachedAt, revokedAt );
 	const domain = siteUrl && ! isPressableLicense ? getUrlParts( siteUrl ).hostname || siteUrl : '';
 
-	const isClientLicense = isAutomatedReferralsEnabled && referral;
+	const isClientLicense = referral;
```

Comment:
[Nitpick] Maybe we could get rid of `isClientLicense` and use `referral` directly?

Reply:
This has been applied. Thanks for the suggestion 👍